test/cheerio.js

```diff
@@ -369,4 +376,33 @@ describe('cheerio', function () {
       });
     });
   });
+  describe('util functions', function () {
+    it('camelCase function test', function () {
+      expect(utils.camelCase('cheerio.js')).toBe('cheerioJs');
+      expect(utils.camelCase('camel-case-')).toBe('camelCase');
+      expect(utils.camelCase('__directory__')).toBe('_directory_');
+      expect(utils.camelCase('_one-two.three')).toBe('OneTwoThree');
+    });
+
+    it('cssCase function test', function () {
+      expect(utils.cssCase('camelCase')).toBe('camel-case');
+      expect(utils.cssCase('jQuery')).toBe('j-query');
+      expect(utils.cssCase('neverSayNever')).toBe('never-say-never');
+      expect(utils.cssCase('CSSCase')).toBe('-c-s-s-case');
+    });
+
+    it('cloneDom : should be able clone single Elements', function () {
+      var result = [];
+      utils.domEach(cheerio('<p>A</p><p>B</p>'), function (i, el) {
+        result = result.concat(utils.cloneDom(el));
+      });
+      expect(result).toHaveLength(2);
```

Comment:
Can we check the actual shape of the response here?

Reply:
I wanted test it with toEqual but because circular reference it will hang jest, so I removed it. But I added some tests.